wcfapp/README.md

```diff
@@ -0,0 +1,59 @@
+# WCF Docker Container Samples
+These samples demonstrate how to dockerize WCF services, either IIS-hosted or self-hosted. A simple "hello world" service contract is used in all samples for both HTTP and NET.TCP transport bindings.
+
+## Prepare Your Environment
+After cloning or downloading this repository, build the solution in the `wcfapp` folder either in Visual Studio or using command line `msbuild wcfapp.sln` in a VS console window. You will need environment and software below installed to complete these samples.
+
+* [Windows 10](https://www.microsoft.com/en-us/windows/get-windows-10) (or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server)) with **'Containers'** feature enabled.
+* [Docker for Windows](https://docs.docker.com/docker-for-windows/) with **'Switch to Windows Containers...'** by right clicking on the Docker whale icon in system tray
+* [Visual Studio 2017](https://www.visualstudio.com/vs/)
+
+## Run WCF Service in Container
+WCF service is supported on .NET Framework, which can run in Windows Server Core based containers. For simplicity, we disabled security in these samples.
+
+### Build a Container with IIS-hosted WCF Service
+Project `WcfServiceWebApp` is created from 'WCF Service Application' template in Visual Studio. A [Dockerfile](WcfServiceWebApp/Dockerfile) is added to the project. We use `microsoft/wcf` image as the base image, which has both HTTP and NET.TCP protocols enabled in IIS and exposes ports 80 (for HTTP) and 808 (for NET.TCP) for the container. We use WCF image with tag `4.6.2` for .NET Framework 4.6.2 in this example, but you can change it to use other tags (eg. `microsoft/wcf:4.7` with tag `4.7`) for WCF images with different versions of .NET Framework. The complete list of [WCF image tags](https://hub.docker.com/r/microsoft/wcf/tags/) can be found from Docker Hub. 
```

Comment:
consider add something like: you can create the dockerfile in the project by add "Docker Support" to the project in Visual Studio 2017

Reply:
I won't mention the Add Docker Support (ADS) in this example, because it will set the user on a different path. The Dockerfile added by ADS could be different from the Dockerfile we are using here. In addition, the ADS for WCF is not yet released.

We certainly want to promote ADS at some point. I am thinking about a blog post or something.

Reply:
Sounds good